### PR TITLE
Align chart to upstream

### PR DESCRIPTION
**Pull request by ubergesundheit**
This is the upstream alignment PR for giantswarm/roadmap#2000

For this, the Giant Swarm chart has been removed and the upstream chart has been imported.

Notes for reviewers
- Please to emphasise revieweing changes to `CHANGELOG.md`, `CONTRIBUTING.md` and `docs/upgrading.md`
- The actuall diff between upstream and our chart can be looked at [here](https://github.com/cert-manager/cert-manager/compare/master...giantswarm:cert-manager-upstream:master)
- The `cert-manager-giantswarm-clusterissuer` got a completely new values schema. Much less reliant on `global` values.
- I tried to implement Giant Swarm specific requirements (for example Monitoring annotations, registry domain handling etc) using changes to `values.yaml`

### CODE REVIEW

**Review by ced0ps — COMMENTED** [review]
LGTM

File: docs/upgrading.md [hunk 1]
```diff
@@ -65,7 +88,7 @@ Note: the provided [migration script](../files/migrate-v090-to-v200.sh) can be u
 helm --tiller-namespace giantswarm delete --purge cert-manager
```
Comment (ced0ps) [inline docs/upgrading.md:88]: Does this still apply? This works with helm 3 and without tiller, right?

File: docs/upgrading.md [hunk 2]
```diff
@@ -2,30 +2,30 @@
 
 ## Upgrading from >= v2.24.0 to v3.0.0
 
-Chart version 3.0.0 and above changed the schema of the chart values. It also changed names of deployments.
+In Chart version 3.0.0 and above the `values.yaml` schema and deplyment names have been changed.
```
Comment (ubergesundheit) [inline docs/upgrading.md:5]:
```suggestion
In Chart version 3.0.0 and above the `values.yaml` schema and deployment names have been changed.
```

File: docs/upgrading.md (hunk 1 above)
Comment (ubergesundheit) [reply docs/upgrading.md:88]: This is not scope of this PR.. let’s ignore this

**Review by ced0ps — COMMENTED** [review]
LGTM

**Review by ced0ps — APPROVED** [review]
LGTM

**Review by ced0ps — APPROVED** [review]
LGTM

**Pull request merged**